contracts/bridge/BridgeTransferERC721.sol

```diff
@@ -66,7 +66,7 @@ contract BridgeTransferERC721 is BridgeTokens, IERC721BridgeReceiver, BridgeTran
         if (modeMintBurn) {
             require(ERC721MetadataMintable(_tokenAddress).mintWithTokenURI(_to, _tokenId, _tokenURI), "mint failed");
         } else {
-            IERC721(_tokenAddress).safeTransferFrom(address(this), _to, _tokenId);
+            IERC721(_tokenAddress).transferFrom(address(this), _to, _tokenId);
```

Comment:
If your change point is applied, onERC721Received() won't be executed even for a normal contract. So I'm concerned that there will be a side effect. Because if we simply use transferfrom(), then we can't check whether `to` address can receive token or not. 
Is it intended?

Reply:
I know that's why the `onERC721Received()` stands for further treatment when a token receives. This further treatment is not available in the receiver address is not a token contract address. If the transaction signer is the sender of that transaction, we do not need to make this change. However, the problem of the current value transfer design is that the handle transaction of value transfer is signed by the operator on each chain. So, it seems inevitable change.

Reply:
Okay. There's another way to confirm whether we received ERC721 well or not, 
But we never know which contract is malicious or not. 
So this can be the proper solution